setup.py

```diff
@@ -64,187 +78,193 @@ class BinaryDistribution(Distribution):
     def is_pure(self):
         return False
 
-def check_for_spice():
-    if not os.path.exists(cspice_dir):
-        import getspice
-        message = 'Unable to find CSPICE at {0}. Attempting to Download CSPICE For you:'.format(cspice_dir)
-        print(message)
-        # Download cspice using getspice.py
-        getspice.getSpice()
-        if not os.path.exists(cspice_dir):
-            message = 'Unable to find CSPICE at {0}. Exiting'.format(cspice_dir)
-            sys.exit(message)
 
+class InstallSpiceyPy(install):
+    """Class that extends the install command and encapsulates the
+    process for installing the required CSPICE distribution at the
+    right place.
+    """
+
+    def run(self):
+        self.check_for_spice()
+    
+        print("Host OS: {}".format(host_OS))
```

Comment:
think that the format without an index number is python 3 specific, or something like that (this why every usage I have includes the number)